intermittent_tracker/templates/most_flaky.html

```diff
@@ -24,22 +24,23 @@ <h2>Flakiest Tests</h2>
                     <th>#</th>
                     <th>Test Path</th>
                     <th>Number of Unexpected Results</th>
-                    <th>GitHub Link</th>
                 </tr>
             </thead>
             <tbody>
                 {% for test in tests %}
                 <tr>
                     <td>{{ loop.index }}</td>
-                    <td>{{ test.path }}</td>
-                    <td>{{ test.total_unexpected_count }}</td>
                     <td>
+                        {{ test.path }}
                         {% if test.issues %}
-                            <a href="https://github.com/servo/servo/issues/{{ test.issues.issues[0].number }}" rel="noopener" target="_blank">issue #{{test.issues.issues[0].number}}</a>
-                        {% else %}
-                            No issues found
+                        {% for issue in test.issues.issues %}
+                        <a href="https://github.com/servo/servo/issues/{{ issue.number }}" rel="noopener" target="_blank">
+                            (#{{ issue.number }})
+                        </a>
+                        {% endfor %}
```

Comment:
This will produce links with the the text of "(#123)" for every bug. Would it be possible to use join to create a list of bugs in parenthesis like the example I showed above. I think you can do this via the Jinja template. Here are some examples:

https://stackoverflow.com/questions/2061439/string-concatenation-in-jinja

To be clear this would look like this:

 `/css/path/to/test/file.html` ([#123](link.html), [#123](link.html), [#123](link.html))